catalog/ui/search-ui/search-redirect/src/main/resources/OSGI-INF/blueprint/blueprint.xml

```diff
@@ -14,20 +14,21 @@
 -->
 
 <blueprint xmlns:cm="http://aries.apache.org/blueprint/xmlns/blueprint-cm/v1.1.0"
+           xmlns:ext="http://aries.apache.org/blueprint/xmlns/blueprint-ext/v1.0.0"
            xmlns="http://www.osgi.org/xmlns/blueprint/v1.0.0">
 
-    <service interface="javax.servlet.Servlet" depends-on="redirectConfiguration">
+    <ext:property-placeholder/>
```

Comment:
❓ Does this allow for the use of system properties within blueprint?

Reply:
Yep. In this case, it isn't necessarily because the properties if the defaultUri will be resolved in the constructor, but I think it's safer to resolve in the blueprint as well.